mersenne-31/src/x86_64_avx512/mod.rs

```diff
@@ -2,3 +2,4 @@ mod packing;
 mod poseidon2;
 
 pub use packing::*;
+pub(crate) use poseidon2::*;
```

Comment:
Nit: newline at the end of the file. (You can set Sublime to do this for you.)

<img width="466" alt="Screenshot 2024-09-26 at 11 06 16" src="https://github.com/user-attachments/assets/ced3187c-53b1-4962-87bf-592cfab57a3a">

Reply:
Yep, my bad. Forgot to run Cargo fmt